.github/workflows/run_tests.yml

```diff
@@ -0,0 +1,58 @@
+name: Run Unit Tests
+
+# Remove this for now, circleci is getting in the way with tests failing
```

Comment:
Is this comment still relevant? Is it saying that a failure from CircleCI caused the GitHub Actions builds to fail? Seems like this may be resolved but I'm unsure.

Reply:
It's not needed anymore, I'll remove this.